app/controllers/application_modal_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ApplicationModalController < ApplicationController
```

Comment:
Will helpers and partials allow us to change the content of the modal dynamically without the need to reload the page?